source/less/components/attractions.less

```
@@ -0,0 +1,26 @@
+.attractions__title {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 26px;
+  color: #000000;
+  margin: 0;
+  margin-bottom: 20px;
+}
+
+.attractions__list {
+  margin: 0;
+}
+
+.attractions__item {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 26px;
+  color: #000000;
+  text-transform: none;
+  margin: 0;
+  margin-bottom: 16px;
+}
+
+.attractions__item:last-child {
+  margin-bottom: 24px;
+}
```

Если ничего не изменилось в кодгайде академии, то элементы типа :last-child мы пишем через &

```suggestion
.attractions__item {
  font-weight: 700;
  font-size: 18px;
  line-height: 26px;
  color: #000000;
  text-transform: none;
  margin: 0;
  margin-bottom: 16px;
  
  &:last-child {
  margin-bottom: 24px;
  }
}
```

Это во-первых,
во-вторых, отступ между секциями должен заваться через секции, а не через последний элемент секции